components/patient-data/ui/src/main/resources/PhenoTips/UIX_Field__charts.xml

```diff
@@ -116,9 +116,13 @@
 #set ($targetClassName = 'PhenoTips.MeasurementsClass')
 #if ($xcontext.action != 'export' &amp;&amp; ($xcontext.action == 'edit' || $doc.getObject($targetClassName)))
   (% class="growth-charts-section" %)(((
-    === $services.localization.render('phenotips.UIXField.growthCharts.heading') ===
-
-    #__expandTools()
+    #if($xcontext.action != 'view')
+      (% class="expandable" %)
+      === $services.localization.render('phenotips.UIXField.growthCharts.heading') ===
```

Comment:
Since this heading is present in both branches of the `#if`, why not put it only once before the test?